frontend_lib/src/component/Timeline/Comment.jsx

```diff
@@ -10,130 +10,180 @@ import EmojiReactions from '../../container/EmojiReactions.jsx'
+                  className={classnames(`${props.customClass}__body__content__text`, 'comment__body__content__text')}
                 >
-                  <HTMLContent isTranslated={props.translationState === TRANSLATION_STATE.TRANSLATED}>{props.text}</HTMLContent>
+                  {(isFile
+                    ? (
+                      <CommentFilePreview
+                        apiUrl={props.apiUrl}
+                        apiContent={props.apiContent}
+                        isPublication={props.isPublication}
+                      />
+                    )
+                    : (
+                      <HTMLContent isTranslated={props.translationState === TRANSLATION_STATE.TRANSLATED}>
+                        {props.text}
+                      </HTMLContent>
+                    )
+                  )}
                 </div>
               </div>
               <LinkPreview apiUrl={props.apiUrl} findLinkInHTML={props.text} />
             </div>
-
-            {props.allowCommentActions && (
-              <DropdownMenu
-                buttonCustomClass='comment__body__content__actions'
-                buttonIcon='fas fa-ellipsis-v'
-                buttonTooltip={props.t('Actions')}
-              >
-                <IconButton
-                  icon='fas fa-fw fa-pencil-alt'
-                  intent='link'
-                  key='editComment'
-                  mode='dark'
-                  onClick={props.onClickEditComment}
-                  text={props.t('Edit')}
-                  title={props.t('Edit comment')}
-                />
-
-                <IconButton
-                  icon='far fa-fw fa-trash-alt'
-                  intent='link'
-                  key='deleteComment'
-                  mode='dark'
-                  onClick={props.onClickDeleteComment}
-                  text={props.t('Delete')}
-                  title={props.t('Delete comment')}
-                />
-              </DropdownMenu>
-            )}
-          </div>
-          <div
-            className={classnames(`${props.customClass}__footer`, 'comment__footer')}
-          >
-            <TranslateButton
-              translationState={props.translationState}
-              onClickTranslate={props.onClickTranslate}
-              onClickRestore={props.onClickRestore}
-              dataCy='commentTranslateButton'
-            />
-            <EmojiReactions
-              apiUrl={props.apiUrl}
-              loggedUser={props.loggedUser}
-              contentId={props.contentId}
-              workspaceId={props.workspaceId}
-            />
           </div>
         </div>
+        <div
+          className={classnames(`${props.customClass}__footer`, 'comment__footer')}
+        >
+          <TranslateButton
+            translationState={props.translationState}
+            onClickTranslate={props.onClickTranslate}
+            onClickRestore={props.onClickRestore}
+            dataCy='commentTranslateButton'
+          />
```

Comment:
Should we have a TranslateButton if isFile?

Reply:
No. It is actually not shown, but you are right, I'll add an explicit `!isFile` :-)

Reply:
fixed